chapters/ch57.md

```diff
@@ -48,4 +48,4 @@ Cuối cùng, nếu bạn không nghĩ rằng pipeline như một chỉnh thể
 
```

Comment:
pipeline as a whole --> mình nghĩ ám chỉ chất lượng cuối cùng của toàn bộ pipline. Dịch hiện tại "như một chỉnh thể" có vẻ chưa đúng ý lắm.

Reply:
Có đề xuất nào không @ngcthuong?

Reply:
Mình dịch là "hệ thống" được không anh? "pipeline as a whole" tức là toàn bộ pipeline riêng lẻ kết hợp với nhau mà như vậy thì sẽ hình thành một hệ thống.

Reply:
Mình sẽ sửa và merge PR này trong vòng 24h nữa.

Reply:
Sửa thành như thế này thì sao mọi người:
"Cuối cùng, nếu bạn nghĩ rằng pipeline hoàn chỉnh sẽ không đạt được chất lượng mức con người"

Reply:
@lkhphuc anh dịch là "hoàn chỉnh" theo em rồi.